atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -374,6 +330,34 @@ def update_repos_from_composes(self):
             self.log.info('adding repo file from compose: %s', compose_info['result_repofile'])
             self.repos.append(compose_info['result_repofile'])
 
+    def _add_filesystem_to_dockerfile(self, file_path, build_dir: BuildDir):
+        """
+        Put an ADD instruction into the Dockerfile (to include the filesystem
+        into the container image to be built)
+        """
+        content = 'ADD {0} /'.format(file_path)
+        lines = build_dir.dockerfile.lines
+
+        custom_image_from = 'FROM koji/image-build'
+        scratch_from = 'FROM scratch'
+
+        for index, line in enumerate(lines):
+            if f'{custom_image_from}\n' not in line and f'{custom_image_from} ' not in line:
+                continue
+            lines[index] = lines[index].replace(custom_image_from, scratch_from)
```

Comment:
In the change_from_in_df plugin, we use the `parent_images` property to set parent images. Maybe this plugin could use a similar approach?